server.py

```diff
@@ -0,0 +1,23 @@
+from twisted.web.server import Site
+from twisted.web.resource import Resource
+from twisted.internet import reactor, endpoints
+
+class Home(Resource):
+    isLeaf = True
+
+    def __init__(self):
+        Resource.__init__(self)
+
+    def render_GET(self, request):
+        path = "/var/www/html/books.toscrape.com/"
+        filepath = '/'.join(request.postpath)
+        fname = path + filepath
+        f = open(fname)
```

Comment:
It's better to open and explicitly close the file using ``with`` statement - e.g. for PyPy this would likely not work due to opening a lot of files and not closing them right away (because the GC is not based on reference counting: http://doc.pypy.org/en/latest/cpython_differences.html#differences-related-to-garbage-collection-strategies)